.github/workflows/reviewdog.yml

```diff
@@ -0,0 +1,18 @@
+name: reviewdog
+on:
+  push:
+    branches:
+      - main
+  pull_request:
```

Comment:
I *think* this should be `pull_request_target` such that this action can run on PRs from forks of the repo. If that's what you want :)